src/Domain/Entities/GameServers/Map.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Crpg.Domain.Entities.GameServers;
+public class Map
+{
+    public int Id { get; set; }
+    public string? Name { get; set; }
+    public int GameServerConfigId { get; set; }
+    public GameServerConfig? GameServerConfig { get; set; }
+}
```

Comment:
I do not understand why the database needs to be aware of what a map is? Also what benefit do we get from this?

Reply:
The most important reason why I followed this path was getting a builder [error.](https://prnt.sc/TPlvJWsPsBKa
)

If you remember, the old property looked like this: `public List<string> Maps { get; set; } = new List<string>();`
Occurs a error when a property of type is used and a primary key for this property is not specified. Entity Framework expects any database table to have a primary key.
Maps property changed to List<Map>. In this case, Entity Framework uses the Map class to detect this property as a data table. Even if I use Fluent API, I was able to find a solution to overcome this problem in this way.

I also think that this way will make it easier to manage server maps from the panel and facilitate log system integration.